website/docs/r/resource_group.html.markdown

```diff
@@ -31,7 +31,7 @@ The following arguments are supported:
     Azure subscription.
 
 * `location` - (Required) The location where the resource group should be created.
-    For a list of all Azure locations, please consult [this link](http://azure.microsoft.com/en-us/regions/).
+    For a list of all Azure locations, please consult [this link](http://azure.microsoft.com/en-us/regions/) or run `az account list-locations`.
```

Comment:
It appears it's possible to output this data as a Table rather than a JSON Blob using the ` --output table` flag - can we update this so that it's more readable:

```
$ az account list-locations --output table
DisplayName            Latitude    Longitude  Name
-------------------  ----------  -----------  ------------------
East Asia               22.267      114.188   eastasia
Southeast Asia           1.283      103.833   southeastasia
Central US              41.5908     -93.6208  centralus
East US                 37.3719     -79.8164  eastus
East US 2               36.6681     -78.3889  eastus2
West US                 37.783     -122.417   westus
North Central US        41.8819     -87.6278  northcentralus
South Central US        29.4167     -98.5     southcentralus
North Europe            53.3478      -6.2597  northeurope
West Europe             52.3667       4.9     westeurope
Japan West              34.6939     135.502   japanwest
Japan East              35.68       139.77    japaneast
Brazil South           -23.55       -46.633   brazilsouth
Australia East         -33.86       151.209   australiaeast
Australia Southeast    -37.8136     144.963   australiasoutheast
South India             12.9822      80.1636  southindia
Central India           18.5822      73.9197  centralindia
West India              19.088       72.868   westindia
Canada Central          43.653      -79.383   canadacentral
Canada East             46.817      -71.217   canadaeast
UK South                50.941       -0.799   uksouth
UK West                 53.427       -3.084   ukwest
West Central US         40.89      -110.234   westcentralus
West US 2               47.233     -119.852   westus2
Korea Central           37.5665     126.978   koreacentral
Korea South             35.1796     129.076   koreasouth
France Central          46.3772       2.373   francecentral
France South            43.8345       2.1972  francesouth
```